index.html

```diff
@@ -12,9 +12,34 @@
 </head>
 
 <body>
+
+   
     <div class="container">
    
         <section class="top-content">
+
+            <header>
+                <nav>
+                    <div class="logo">
+                        <img src="img/lambda-black.png" alt="Lambda">
+                    </div>
+                    <div class="links">
+                    <a href="../web-Sprint-Challenge-User-Interface-and-Git/index.html">Home</a>
```

Comment:
This will actually break if you use the links.  When adding links to other pages in relation to where the file you adding the link to resides.  In this case index.html resides in the same folder as about.html so you can just say about.html and index.html instead of ../web-Sprint...../about.html.